config/application.rb

```diff
@@ -14,6 +14,12 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    require 'indexer'
+    require 'pid_gatherer'
+    require 'bulk_reindexer'
+    require 'profiler'
```

Comment:
nope, profiler's useful in any environment.  we originally only included the gem in the development bundle, but it's always in, now.